news/6340.feature

```diff
@@ -0,0 +1,3 @@
+Added a new option `--save-wheel-names` to command `wheel`.
```

Comment:
Note that this file uses the [rst](http://docutils.sourceforge.net/rst.html) syntax. AFAIK it needs double backtick here.